.github/workflows/clone_with_pinned_version.yml

```
@@ -0,0 +1,49 @@
+name: JEDI Bundle Pinned Version Clone Test
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+
+jobs:
+  buildbundle:
+    name: Clone oops bundle
+    runs-on: ubuntu-latest
+
+    steps:
+
+    # Setup Python
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    # Update conda
+    - name: Update conda
+      run: conda update -n base -c defaults conda
+
+    # Install pip
+    - name: Install pip
+      run: conda install pip
+
+    # Clone the code repo
+    - name: Clone code repo
+      uses: actions/checkout@v2
```

Per the [docs](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python), updating to a more recent version of this.

```suggestion
      uses: actions/checkout@v4
```